addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -159,6 +159,9 @@ class MessageList extends Component {
                     // position
                     this._adjustScrollForExtraMessagesAtTheStart();
                     break;
+                case 'post-new-message':
```

Comment:
move to case with others that do `_adjustScrollForExtraMessagesAtTheEnd` + rename to `new-message-posted` or something similar to how other names are built